### PR TITLE
Mint page UI

### PR DESCRIPTION
**Pull request by johnmosesman**

### CODE REVIEW

**Comment by ryanlabouve** [comment]
Screenshot + Tests?

**Comment by ryanlabouve** [comment]
(that was a joke)

File: src/lib/header/Header.svelte [hunk 1]
```diff
@@ -12,8 +13,7 @@
 	{#if $user.walletAddress}
 		<div class="relative cursor-pointer" on:mouseenter={() => (showDisconnect = true)}>
 			<div class="px-2 lonk pretty-lonk">
-				Connected as {$user.walletENSAddress ||
-					`${$user.walletAddress.slice(0, 2)}...${$user.walletAddress.slice(-4)}`}
+				Connected as {$user.walletENSAddress || prettyAddress($user.walletAddress)}
```
Comment (ryanlabouve) [inline src/lib/header/Header.svelte:16]: Would you mind reverting this? Have a change here in the another pr

File: src/lib/nft-thumbnail.svelte [hunk 2]
```diff
@@ -22,7 +22,7 @@
 		token_id = token_uri.split('/').slice(-1).join(''); // TODO: do this less terribly
 
 		evaluation = await evaluateNft(token_uri);
-		grade = evaluation[0].toLowerCase();
```
Comment (ryanlabouve) [inline src/lib/nft-thumbnail.svelte:25]: -3 points

File: src/lib/utils/functions.ts [hunk 3]
```diff
@@ -177,3 +177,47 @@ export async function getImageType(imageValue) {
     }
   }
 }
+
+export function getURLFromURI(uri) {
+  // this code is adapted from CheckMyNFT
+  const ipfsGateway = 'https://ipfs.moralis.io:2053/ipfs/';
+  try {
```
Comment (ryanlabouve) [inline src/lib/utils/functions.ts:184]: Can we just remove this try?

**Review by ryanlabouve — DISMISSED** [review]
This is perfect % all the comments but merge away

**Comment by ryanlabouve** [comment]
After https://github.com/ryanlabouve/forever721/pull/43 merges, we may rebase and do some quick QA

File: src/lib/header/Header.svelte (hunk 1 above)
Comment (johnmosesman) [reply src/lib/header/Header.svelte:16]: The whole function is just this file?

File: src/lib/nft-thumbnail.svelte (hunk 2 above)
Comment (johnmosesman) [reply src/lib/nft-thumbnail.svelte:25]: +3 points since I removed it

**Pull request merged**